src/Main.elm

```diff
@@ -179,7 +183,7 @@ view model =
             , numberButton CalNumEight
             , numberButton CalNumNine
             , numberButton CalNumZero
-            , button [ class "clearAll" ] [ text "AC" ]
+            , button [ class "clearAll", onClick AllClear ] [ text "AC" ]
```

Comment:
I think you should update `class "clearAll"` to `class "allClear"`. Having consistent nomenclature will help in the long run!